test/test_pairs_classifiers.py

```diff
@@ -133,10 +133,25 @@ def fit(self, pairs, y):
     pairs, y = self._prepare_inputs(pairs, y,
                                     type_of_inputs='tuples')
     self.components_ = np.atleast_2d(np.identity(pairs.shape[2]))
-    self.threshold_ = 'I am not set.'
+    # self.threshold_ is not set.
     return self
 
 
+def test_unset_threshold():
+  # test that set_threshold indeed sets the threshold
```

Comment:
this description is wrong

Reply:
Addressed in PR #269, sorry for the mishap.